### PR TITLE
mens.tasclap.jp

### PR DESCRIPTION
**Pull request by SKEIDs**
1 - https://github.com/AdguardTeam/AdguardFilters/issues/102785

2 - "Recommend" button is an ad to introduce the site.
so, I think it should be removed.
image

### CODE REVIEW

**Comment by Yuki2718** [comment]
> "Recommend" button is an ad to introduce the site.

No, the button leads to their articles and should not be removed even though the articles are some kind of native-ad. Please understand that our filter lists are for everyone including average Joe and Jane but not solely for adblocking or IT geeks.

**Comment by SKEIDs** [comment]
> > "Recommend" button is an ad to introduce the site.
> 
> 
> 
> Wait, I thought you added to JPF but actually to Annoyances. That may be okay.


I am amateur. 
I don't think my judgment can be trusted.

**Comment by Yuki2718** [comment]
Ha-ha, actually in this case probably you're right and I was wrong, just my mistook.

**Comment by SKEIDs** [comment]
> Ha-ha, actually in this case probably you're right and I was wrong, just my mistook.

ok

**Comment by Yuki2718** [comment]
Just change `##.l-action-button` to `mens.tasclap.jp##.l-action-button-recommend` to minimize false positive, then everything looks fine.

**Comment by SKEIDs** [comment]
> Just change `##.l-action-button` to `##.l-action-button-recommend` to minimize false positive, then everything looks fine.

I see. finished

**Review by Yuki2718 — APPROVED** [review]

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -736,6 +736,7 @@ logitech.com#$#.email-subscription-modal { display: none !important; }
 flipboard.com##.anchored-dialog.signup-cta
 digify.com###elementor-popup-modal-7782
 mens.tasclap.jp##.c-notice
+mens.tasclap.jp##.l-action-button
```
[the remark this answers is not in the document]
Comment (SKEIDs) [reply AnnoyancesFilter/sections/popups.txt]: ok.done.
I am sorry. This pull request moved #103242

**Pull request closed without merging**